.circleci/config.yml

```diff
@@ -130,3 +159,7 @@ workflows:
       - {test-11_check-iso-work-fol: {requires: [build]}}
       - {test-11_check-fol: {requires: [build]}}
       - {test-11_check-failure: {requires: [build]}}
+
```

Comment:
With these 3, I guess we're hitting our concurrency limit on CircleCI, so some tests wait until the others are finished.

I tried merging these 3 into a single step, then the whole step takes really long time.

So, I'm open for suggestions here.